src/chrome/content/rules/123systems.xml

```diff
@@ -1,4 +1,9 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://123systems.net/ => https://123systems.net/: Too many redirects while fetching 'https://123systems.net/'
+Fetch error: http://www.123systems.net/ => https://www.123systems.net/: Too many redirects while fetching 'https://www.123systems.net/'
+
```

Comment:
You must be reading a different revision, since the version github presents to me and my local version have this:

`/+(?!billing(?:$|[?/]))`

Which for our purposes means 'anything but "/billing"'.